src/components/TodoProgress.jsx

```diff
@@ -0,0 +1,221 @@
+import styled from "styled-components";
+import TodoListItem from './TodoListItem';
+import { useTodoContext } from './TodoContext';
+
+const TodoProgressStyle=styled.div`
+position:relative;
+    .todoProgressContainer{
+        border:1px solid white;
+        position:relative;
+        background-color: #262626;
+        border-radius: 8px;
+        width:370px;
+        display:flex;
+        flex-direction:column;
+        flex:1;
+        padding:0px 20px 20px 20px;
+        align-items:center;
+        margin:20px;
+        height:800px;
+        overflow-y:auto;
+    }
+    .todoProgressContainer::-webkit-scrollbar {
+        display: none;
+      }
+    .todoProgressTitle{
+        position:sticky;
+        top:0px;
+        background-color: #262626;
+        z-index:1;
+        color:white;
+        font-size:37px;
+        font-weight:700;
+        text-align: left;
+        width:100%;
+        padding:20px;
+        margin-bottom:5px;
+
+    }
+    .todoListPlus{
+        position:absolute;
+        width:26px;
+        height:26px;
+        background-color:white;
+        transform:translate(-50%,0%);
+        left:50%;
+        bottom:5px;
+        border-radius:50%;
+
+    }
+    .todoListItem{
+        position:relative;
+        display:flex;
+        flex-direction:column;
+        margin:15px;
+        padding:10px;
+        border-radius:8px;
+        background-color:#D93535;
+        width:90%;
+        color:white;
+        &.red {
+            background-color: #D93535;
+        }
+
+        &.blue {
+            background-color: #307FE2;
+        }
+
+        &.green {
+            background-color: #00B294;
+        }
+
+        &.purple {
+            background-color: #7678D1;
+        }
+    }
+    .todoListItemTitle{
+        font-size:20px;
+
+        margin:30px 10px 10px 10px;
+    }
+    .todoListItemContent{
+        margin:10px;
+    }
+    .todoListItemXBtn{
+        position:absolute;
+        width:25px;
+        height:30px;
+        top:0px;
+        right:10px;
+        text-align:center;
+    }
+
+    .todoListItemWriteBtn{
+        position:absolute;
+        right:35px;
+        top:5px;
+        width:25px;
+        height:25px;
+        text-align:center;
+    }
+    .todoListEditContainer{
+        display:flex;
+        flex-direction:column;
+        position:relative;
+    }
+    .todoListEditTitle{
+        border-radius:8px;
+        border:1px white solid;
+        background-color: rgb(200, 200, 200,0.5);
+        margin:2px;
+        font-size:20px;
+        width:50%;
+        color:white;
+        font-style:bold;
+    }
+    .todoListEditTitle::placeholder{
+        color:white;
+        font-style:bold;
+    }
+    .todoListEditContent{
+        border-radius:8px;
+        border:1px white solid;
+        background-color: rgb(200, 200, 200,0.5);        
+        margin:2px;
+        color:white;
+    }
+    .todoListEditContent::placeholder{
+        color:white;
+
+    }
```

Comment:
styled-coponent에서는
```suggestion
    .todoListEditContent{
        border-radius:8px;
        border:1px white solid;
        background-color: rgb(200, 200, 200,0.5);        
        margin:2px;
        color:white;
    }
    &::placeholder{
        color:white;
    }
```
처럼도 사용할 수 있어요!
(&뒤에 콜론이 1개였는지 2개였는지는 기억이 나질않습니다 ㅜ)